### DISCUSSION TITLE
请教一下，Table如何不分页一次加载全部数据？

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2278] [讨论]
有一些表体积不大，而且API也可能不支持分页参数，如何一次加载全部数据？

**Comment by HuangLei0x2e5** [comment]
我自己研究出来一个办法：
表格加载的时侯调用
  layui.table.render(parameters);
parameters 就是那些属性，但是不要设置url， 也就是说只加载格式，不加载数据

然后自己写一个刷新方法，拿到数据后调用
  layui.table.reloadData(id, {data: responseData});
  
  responseData是后端返回的整张表的数据
  
 
这样拿数据的方法适合数据量不大的表，更简单更省事

**Reply by HuangLei0x2e5** [reply]
再多嘴一句，layui设计的功能很强大，但是文档有的地方实在是过于复杂和书面化
对于前端来说可能很多东西是常识，但对于我们这些搞后端的半瓶水们，很多思维方式和习惯不一样，理解起来比较困难
如果能多一些这种简单的例程，可能会节省大家很多时间

**Reply by sentsim** [reply]
这些其实也不算是常识，而是只要认真看了文档，应该都能找到解决方案。
Layui 的文档我认为已经算是通俗易懂的了。
